ChangeLog

```diff
@@ -1,3 +1,8 @@
+2021-05-20  Ilya Zarubin  <ilya.a.zarub@gmail.com>
+
+	* inst/include/history.h: add facilities for tracking of ancestral lines (mostly additions to the historyelement class)
```

Comment:
We've been restricting the line lengths to I think around 80 characters/columns for readability, so these two dot points could be split across more lines. E.g. see your 2021-05-20 src/nonLinPMMH.cpp item.

Adam and Dirk use Emacs where this is all automatic.